pytest.ini

```diff
@@ -1,6 +1,4 @@
 [pytest]
-# always check coverage of janitor.
-addopts = --cov=janitor --cov-report term-missing --cov-report xml --durations=0 --doctest-modules --doctest-glob="janitor/*.py" --ignore-glob="nbconvert_config.py" -v -n auto --color=yes
```

Comment:
I don't like `addopts`. That will force all running to use that configuration.
If I only want to test `tests/test_df_registration.py`, I do not need `pytest-xdist` to enable the multi-core feature.

Reply:
I see. This sounds reasonable, though I think we do want to have sane defaults here; can you think of a way to enable:

1. Sane default that includes multi-core,
2. Flexibility to disable certain addopts,
3. Without excessive duplication of commands across files?

Multi-core testing will really speed up testing in the CI and locally.

Reply:
We use addopts because of we are too lazy to key in long commands.
So the solution is Makefile